### ISSUE TITLE
Add celery+redis backend

### ISSUE THREAD

**Issue by eoranged** [Parallels/rq-dashboard#241]

**Comment by justinttl** [comment]
Came across this project when I am looking for a monitoring GUI like redis-insights specifically for `celery[redis]` which seems to use list based redis queues. Is this currently supported in rq-dashboard?

**Comment by eoranged** [comment]
Nope. I don't believe this will be supported any time soon, It was just an idea.
